examples/mnist/eth_mnist.py

```diff
@@ -233,27 +233,7 @@
 
         # Optionally plot various simulation information.
         if plot:
-            # image = batch["image"].view(28, 28)
```

Comment:
Please put these plots back in.

Reply:
The plots were in comments since @coopersigrist change 10 days ago. 
Why they commented?

Reply:
@coopersigrist commented them out while he was doing hyper-parameter tuning, and forgot to un-comment them.